compiler/test-coursier/dotty/tools/coursier/CoursierScalaTests.scala

```diff
@@ -155,7 +155,7 @@ object CoursierScalaTests:
     (code, out.toList)
 
   def csScalaCmd(options: String*): List[String] =
-    csCmd("dotty.tools.MainGenericRunner", options*)
+    csCmd("dotty.dist.MainGenericRunner", options*)
```

Comment:
I really don't think we should break this name.  I can't even understand why the package name was changed.  Why can't it stay as `dotty.tools.MainGenericRunner`?

Reply:
My feedback for Scala 2 is that it is in fact annoying to wonder where is Main after all these years. So putting things in the most obvious path pays dividends.

Reply:
Yeah I did have a bad feeling about this one... 
The `dotty.tools` namespace is used by the `compiler` subproject, and I don't like the idea of two subprojects adding into the same namespace. 

That being said, there's probably dragons waiting if we change the namespace, so I just moved `MainGenericCompiler` and `MainGenericRunner` back to `dotty.tools`.